tests/Database/DatabaseSchemaBuilderTest.php

```diff
@@ -42,6 +45,29 @@ public function testDropDatabaseIfExists()
         $builder->dropDatabaseIfExists('foo');
     }
 
+    public function testDropIndexIfExists()
```

Comment:
Transforming the `dropIndexIfExists` to accept array, you should write a test for that.

Reply:
@devajmeireles new test added.